src/App.jsx

```diff
@@ -41,6 +42,9 @@ export default function App() {
         <Route path="/" element={<Home />}>
           Home
         </Route>
+        <Route path="/technologie" element={<Technologie />}>
```

Comment:
<Route path="/technologie" element={<Technologie />} />

Easier to write it like that I think ! 
Can you remove "zdzq" ?